src/encoded/metadata.py

```diff
@@ -77,10 +81,10 @@ def read(self):
 
 def generate_file_download_header(download_file_name: str):
     """ Helper function that generates a suitable header for the File download """
-    header1 = ['###', 'Metadata TSV Download', '', '', '', '']
+    header1 = ['###', 'Metadata TSV Download', '', '', '', '', '']
     header2 = ['Suggested command to download: ', '', '',
                'cut -f 1 ./{} | tail -n +3 | grep -v ^# | xargs -n 1 curl -O -L '
```

Comment:
Could we update this to use the annotated filename as the download name if it exists, otherwise falling back to the name from S3? My bash skills aren't good enough to suggest a solution, but would want to change the curl command to `curl -o {annotated_filename}` if it's not null.